libs/core/tests/unit_tests/language_models/llms/test_base.py

```diff
@@ -93,6 +93,7 @@ async def test_async_batch_size() -> None:
         assert (cb.traced_runs[0].extra or {}).get("batch_size") == 1
 
 
+@pytest.mark.xfail(reason="This test is failing due to a bug in the testing code")
```

Comment:
The linter found an issue in the test code here.
It was not seen before because eval_response was never executed.

Reply:
This is fixed by #30228 .